server-base.R

```diff
@@ -544,16 +584,9 @@ shinyServer(function(input, output, session){
 
   # Hide/show panels on user-demand
   shinyjs::onclick("toggle_panel", shinyjs::toggle(id = "input_panel", anim = FALSE))
-  shinyjs::onclick("toggle_legend", shinyjs::toggle(id = "zone_legend", anim = FALSE))
+  shinyjs::onclick("toggle_trip_menu", shinyjs::toggle(id = "trip_menu", anim = FALSE))
   shinyjs::onclick("toggle_map_legend", shinyjs::toggle(id = "map_legend", anim = FALSE))
 
-  observe({
-    if (input$map_base == 'IMD')
-      shinyjs::hide("zone_legend")
-    else
-      shinyjs::show("zone_legend")
-  })
```

Comment:
Looks like there is a window clash on IMD
![screenshot from 2016-10-04 21-34-38](https://cloud.githubusercontent.com/assets/6289830/19091455/9d828ebe-8a7a-11e6-9173-8dbc55077393.png)